docs/architecture/adr-076-combine-spec-repo.md

```diff
@@ -0,0 +1,97 @@
+# ADR 076: Combine Spec and Tendermint Repositories 
+
+## Changelog
+
+- 2022-02-04: Initial Draft. (@tychoish)
+
+## Status
+
+Accepted.
+
+## Context
+
+While the specification for tendermint was original in the same
+repository as the Go implementation, at some point the specification
+was split from the core tendermint repository and maintained
+separately from the implementation. While this makes sense in
+promoting a conceptual separation of specification and implementation,
+in practice this separation was a premature optimization apparently
+aimed at supporting alternate implementations of tendermint. 
+
+However, maintaining a spec repo has ended up being a premature
+optimization: there are no current projects to develop alternate
+implementations of tendermint based on the common specification, and
+having separate repositories creates an ongoing maintenance burden to
+support versioning and publication.
+
+## Decision
+
+The specification repository should be merged
+
+Stakeholders including representatives from the maintainers of the
+spec, the go implementation, and the tendermint rust library, agreed
+to merge the repositories in the tendermint core dev meeting on 27
+January 2022.
+
+## Alternative Approaches
+
+The main alternative to this was *do nothing* and continue to maintain
+separate versioning for both repositories, complicating the release
+process for both repositories and creating a difficult to resolve 
```

Comment:
Suggestions to clarify that "do nothing" was relative to merging, not other changes, and finish up an incomplete

```suggestion
The main alternative we considered was to keep separate repositories,
and to introduce a coordinated versioning scheme between the two, so
that users could figure out which spec versions go with which versions
of the core implementation.

We decided against this on the grounds that it would further complicate
the release process for _both_ repositories, without mitigating any of
the other existing issues.
```